kitsune/sumo/static/sumo/less/fonts.less

```diff
@@ -44,6 +44,66 @@
     url('@{fontpath}/Komikax-webfont.ttf?v=1') format('truetype'), /* Safari, Android, iOS */
     url('@{fontpath}/Komikax-webfont.svg?v=1#svgFontName') format('svg'); /* Legacy iOS */
   }
+
+  @font-face {
+    font-family: 'Zilla Slab';
+    font-weight: normal;
+    font-style: normal;
+    font-display: swap;
+    src: url('@{fontpath}/ZillaSlab-Regular.ttf') format('truetype'), /* Safari, Android, iOS */
+    url('@{fontpath}/ZillaSlab-Regular.woff') format('woff'), /* Modern Browsers */
+    url('@{fontpath}/ZillaSlab-Regular.woff2') format('woff2');
```

Comment:
As browsers will use the first supported font format, we should reverse the order here (prioritizing `woff2`).

Reply:
Reversed the order to use woff2 first.